order.go

```diff
@@ -39,32 +39,125 @@ type OrderServiceOp struct {
 	client *Client
 }
 
+type orderStatus string
+
+// https://shopify.dev/docs/api/admin-rest/2023-07/resources/order#get-orders?status=any
+const (
+	//Show only open orders.
+	StatusOpen orderStatus = "open"
```

Comment:
Sorry, one more change.
`StatusOpen` is a bit generic, let's change this to `OrderStatusOpen` to make sure we "namespace" this to just status in case we'll have other objects with a status where we want to do something similar.
(same for all the consts in this block)

You already did this for e.g. FulfillmentStatusShipped so let's do it here as well.

Reply:
@oliver006 I omitted a namespace on purpose because of the genericness.  The `orderStatus` and related `Status...` consts are used for both order and draft orders.  I didn't want `OrderStatusOpen` and `DraftOrderStatusOpen`.  I guess we could just use `OrderStatusOpen` for both, but then the naming isn't consistent.

Let me know your thoughts.  It really doesn't matter either way, both are right and wrong at the same time.

Reply:
Yup, use `OrderStatusOpen` (and the type `orderStatus`) for both.

Reply:
Done